pkg/porter/create_test.go

```diff
@@ -39,5 +42,125 @@ func TestCreate(t *testing.T) {
 	dockerignoreStats, err := p.FileSystem.Stat(".dockerignore")
 	require.NoError(t, err)
 	tests.AssertFilePermissionsEqual(t, ".dockerignore", pkg.FileModeWritable, dockerignoreStats.Mode())
+}
+
+// tests to ensure behavior similiarity with helm create
+
+func TestCreateInDirectory(t *testing.T) {
+	// Create a temporary directory for testing
+	tempDir, err := ioutil.TempDir("", "porter-test")
```

Comment:
```suggestion
	tempDir, err := os.TempDir("", "porter-test")
```
ioutil.TempDir is deprecated and will not pass the gate for staticcheck/go vet